src/govuk/components/exit-this-page/_index.scss

```diff
@@ -2,25 +2,26 @@
 
 @include govuk-exports("govuk/component/exit-this-page") {
   .govuk-exit-this-page {
+    @include govuk-responsive-padding(8, "bottom");
     z-index: 1000;
     top: 0;
     left: 0;
     width: 100%;
```

Comment:
I don't think this is necessary anymore if we're applying `width: 100%` to the button on line 24.

Reply:
Checked and it is indeed not doing much. Have removed.